src/utils/headers.ts

```diff
@@ -177,6 +177,7 @@ export function getHeadersApplicableToAllResources(headers: SecurityHeaders) {
     Object.entries(headers)
     .filter(([key]) => appliesToAllResources(key as OptionKey))
     .map(([key, value]) => ([getNameFromKey(key as OptionKey), headerStringFromObject(key as OptionKey, value)]))
+    .filter(([_, value]) => Boolean(value))
```

Comment:
```suggestion
    .filter(([, value]) => Boolean(value))
```

Addresses CodeQL warning